deploy/backplane/cee/01-cee-cluster-readers-cluster.ClusterRole.yml

```
@@ -39,3 +39,13 @@ rules:
   - get
   - list
   - watch
+# CEE can view machineautoscalerconfigs
+- apiGroups:
+  - autoscaling.openshift.io
+  resources:
+  - '*'
```

```suggestion
  - clusterautoscalers
  - machineautoscalers
```

Been thinking about use of '*', probably best to be explicit generally so any additions in the future are not going to break the assumptions about non-sensitive data.